src/routes/v3Get.integration.ts

```diff
@@ -0,0 +1,42 @@
+import request from 'supertest';
+import { app, server } from '../main';
+import * as Sentry from '@sentry/node';
+import sinon from 'sinon';
+import * as GraphQLCalls from '../graph/graphQLClient';
+
```

Comment:
the integration test simulate and cover error cases, in future we can assert on error headers etc

ETLs of REST->graphQL and graphQL-> REST is primarily covered by spec test